install.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+pip install  --user -r $DIR/requirements.txt
```

Comment:
minor: I believe we could remove this line if we remove `--no-dependencies` further down as the setup declares the dependencies from the `requirements.txt`